### ISSUE TITLE
Task 1: Simple Calculator App

### ISSUE THREAD

**Issue by bhaveshittadwar** [bhaveshittadwar/100-Days#1]

**Comment by Lonecloud7** [comment]
do you need a react calculator app hosted somewhere online somewhere like netlify?

**Comment by rhinoxD** [comment]
Yes please answer, do you need a fully functional calculator app deployed or just a basic setup?

**Comment by bhaveshittadwar** [comment]
Hi @Lonecloud7 , @rhinoxD . Yes and No. I do want a fully functional calculator app, but I am planning to deploy it as a single website with multiple projects, the calculator being the first. Sorry, I will not be using any frontend libraries or frameworks like react for this project, although I am planning to do one soon (Maybe on Svelte). Enough background info, now the real deal. This project is built on Node, and you will find a list of projects on its home page and on the `/projects/<project_name>` route you will find the app for it.

I have already set up the Node Server, configured linting, webpack, scss, and all the frontend mumbo-jumbo. About the calculator, as of now, I have the HTML and barebones CSS ready. I need someone to work on the js logic for it. Is anyone up for it? This will help you understand how frontend assets could be configured on a node serve.

**Comment by rhinoxD** [comment]
Alright, even though I have no experience with nunjucks but I'll still give it a try.
